utils/cookie.js

```diff
@@ -25,14 +25,29 @@ function getSessionDuration() {
 
 /**
  * if valid token will return psychologist data
- * @param {*} token 
+ * @param {*} token
  */
-module.exports.verifyJwt = function verifyJwt(token) {
+const verifyJwt = function verifyJwt(token) {
   try {
     const verified = jwt.verify(token, config.secret);
     return verified;
   } catch (err) {
     console.debug("invalid token");
     return false;
   }
-}
\ No newline at end of file
+}
+module.exports.verifyJwt = verifyJwt
+
+/**
+ *  Get currently logged in psy's id
+ */
+module.exports.getCurrentPsyId = (req) => {
+  // todo(estellecomment) what happens when no cookie/token/uuid ?
+  const jwtToken = req.cookies.token
+  const tokenData = verifyJwt(jwtToken)
+  if (!tokenData.psychologist) {
+    throw new Error('JWT token has no psychologist associated')
+  }
+  const psyUuid = tokenData.psychologist.dossierNumber
+  return psyUuid
```

Comment:
par rapport à ce todo, normalement les controller qui vont appeller la fonction `getCurrentPsyId` ne sont accessible seulement si il y a le token de set

https://github.com/betagouv/sante-psy/blob/main/index.js#L76-L82

le seul cas qu'on peut rencontrer c'est que le token n'est plus valide (validité configurable qui est fixée à 2 heures pour le moment) et qui retournera `false` dans ce cas au lieu de la valeur

https://github.com/betagouv/sante-psy/blob/main/.env.sample#L29

proposition non obligatoire 
```suggestion
module.exports.getCurrentPsyId = (req) => {
  const jwtToken = req.cookies.token
  const tokenData = verifyJwt(jwtToken)
  if (!tokenData) {
    throw new Error('JWT token invalid')
  }
  const psyUuid = tokenData.psychologist.dossierNumber
  return psyUuid
```